server/src/AMQPConnectionInfo.cc

```diff
@@ -32,6 +34,7 @@ using namespace mlpl;
 struct AMQPConnectionInfo::Impl {
 	Impl()
 	: m_URL(),
+	  m_URLBuf(NULL),
```

Comment:
> m_URLBuf is only used in setURL(), isn't it.
> If so, it's unnecessary to be an instance variable.

No, it's used by m_parsedURL.

It's a labyrinth of rabbitmq-c.
This buffer is referred from m_parsedURL (amqp_connection_info).
So we need to keep it until m_parsedURL is destroyed.